src/theme/components/global.ts

```diff
@@ -66,6 +67,11 @@ const selectTextInputDisabledStyles = {
   color: "ui.gray.dark",
   opacity: "1",
 };
+// Used for the disabled state of Select and TextInput
+const selectTextInputFocusStyles = {
+  ...activeFocus(),
+  borderColor: "ui.focus",
+};
```

Comment:
The `radio.ts` uses `activeFocus` but that also has a `borderColor: "ui.focus"` value. So maybe `selectTextInputFocusStyles` can be used there as well.

Or maybe, `borderColor: "ui.focus"` should go into `activeFocus` and `selectTextInputFocusStyles` can be replaced by `activeFocus`.

But did you do this because the `global.ts` file uses `activeFocus` but it doesn't need the `borderColor: "ui.focus"` rule?

Reply:
```But did you do this because the global.ts file uses activeFocus but it doesn't need the borderColor: "ui.focus" rule?```

Yes, exactly.  If `borderColor` was part of `activeFocus`, it would impact `Button` negatively, so I only inlcuded it where it was needed.